.github/workflows/release-nightly.yml

```diff
@@ -1,8 +1,10 @@
 name: Release nightly
 
 on:
-  schedule:
-    - cron: '0 2 * * *' # run at 2 AM UTC
+  push:
+    branches:
+      - main
+      - chore/fix-nightly-builds
```

Comment:
Do we need this branch to trigger the CI?

Reply:
To be honest I'm not sure, I thought of removing it but it was present before so I assumed it might be handy as part of a hot fix workflow. Maybe @ilgooz knows about it and can clarify, otherwise we can remove it. I would be in favor of removing that branch from the list.

Reply:
I removed it in daef149. We can add it again later if necessary.